sycl/test-e2e/README.md

```diff
@@ -343,3 +343,29 @@ Once the issue is created, you can update the test by adding `XFAIL` and
 
 If you add `XFAIL` without `XFAIL-TRACKER` directive,
 `no-xfail-without-tracker.cpp` test will fail, notifying you about that.
+
+## Marking tests as unsupported
+
+Some tests may be considered unsupported, e.g.:
+* the test checks the feature that is not supported by some
+  backend / device / OS / etc.
+* the test is flaky or hangs, so it can't be marked with `XFAIL`.
+
+In these cases the test can be marked with `UNSUPPORTED`. This mark should be
+followed by either `UNSUPPORTED-INTENDED` or `UNSUPPORTED-TRACKER` depending on
+whether the test is not intended to be run with some feature at all or it was
+temporarily disabled due to some issue.
+```
+// UNSUPPORTED: cuda, hip
+// UNSUPPORTED-INTENDED: only supported by backends with SPIR-V IR
+
+// Sporadically fails on DG2.
+// UNSUPPORTED: gpu-intel-dg2
+// UNSUPPORTED-TRACKER: https://github.com/intel/llvm/issues/DDDDD
+// *OR*
+// UNSUPPORTED-TRACKER: PRJ-1234
```

Comment:
Is it an issue number from the internal tracking system?

Reply:
Yes, now we allowed to expose it. But no links :)

Reply:
I'm not sure we can still do that. Can you please consult with our security team, or there was such conversation already?

Reply:
> I'm not sure we can still do that. Can you please consult with our security team, or there was such conversation already?

I've sent to you an internal wiki. We have indeed got an approval to mention IDs of internal trackers in public.